feconf.py

```diff
@@ -414,6 +414,11 @@ def get_empty_ratings():
 REDISHOST = 'localhost'
 REDISPORT = 6379
 
+# Replace project id with the correct project id when switching to the prod
```

Comment:
NOTE TO RELEASE COORDINATORS: Replace this project ID with the ...

Reply:
Done, added it to redis too.